index.html

```diff
@@ -43,7 +43,8 @@
       <a-light type="point" light="color: #fff; intensity:0.6" position="3 10 1"></a-light>
       <a-light type="point" light="color: #fff; intensity:0.2" position="-3 -10 0"></a-light>
       <a-light type="hemisphere" groundColor="#888" intensity="0.8"></a-light>
-      <a-entity id="logo" obj-model="obj: #logoobj; mtl: #logomtl"></a-entity>
+      <a-entity id="logo" obj-model="obj: #logoobj; mtl: #logomtl"
+                animation="dur: 4000; property: components.obj-model.model.children.0.material.opacity; from: 1; to: 0; startEvents: started-painting"></a-entity>
```

Comment:
The way I would do this is creating a `logo-model` component with an opacity property. Code (untested) below:

```
AFRAME.registerComponent('logo-model', {
  schema: {
    opacity: {default: 1.0}
  },
  init: function () {
    this.el.setAttribute('obj-model', 'obj: #logoobj; mtl: #logomtl');
  },
  update: function () {
    this.el.getObject3D('mesh').children[0].material.opacity = this.data.opacity;
  }
});


```

Reply:
Animation component would be:

`animation="dur: 4000; property: logo-model.opacity; from: 1; to: 0; startEvents: started-painting"`